src/core/execute.c

```diff
@@ -730,14 +730,20 @@ static int restore_confirm_stdio(int *saved_stdin, int *saved_stdout) {
         return r;
 }
 
-static int ask_for_confirmation(const char *vc, const char *cmdline, char *response) {
+static int ask_for_confirmation(const char *vc, const Unit *u, const char *cmdline, char *response) {
         int saved_stdout = -1, saved_stdin = -1, r;
 
         r = setup_confirm_stdio(vc, &saved_stdin, &saved_stdout);
         if (r < 0)
                 return write_confirm_error(r, vc);
 
-        r = ask_char(response, "yns", "Execute %s? [Yes, No, Skip] ", cmdline);
+ask_again:
+        r = ask_char(response, "ynvs", "Start %s ? [Yes, No, View, Skip] ", u->id);
```

Comment:
I am not sure I like this patch at all: services can easily have five different ExecStartPre= lines, and we should not ask the same question for all five, over and over again...

The overly long question string is an issue, but it's something I would combat with ellipsize(). I am fine with with adding "v" on top of that though, to see the full string then, plus the service name.